packages/shared/src/buttons/IncrementDecrementButtons.js

```diff
@@ -45,78 +35,60 @@ const Button = ({
 };
 
 export default function IncrementDecrementButtons(props: {|
-  onIncrement: () => void,
-  onDecrement: () => void,
-  number: number,
-  min?: number,
-  max?: number,
+  +onIncrement: () => void,
+  +onDecrement: () => void,
+  +number: number,
+  +showNumber: boolean,
+  +numberStyle?: StylePropType,
+  +min?: number,
+  +max?: number,
 |}) {
   const disableDecrement = props.number === props.min;
   const disableIncrement = props.number === props.max;
 
   return (
-    <View style={styleSheet.buttonsGroup}>
-      <View style={styleSheet.buttonLeft}>
-        <Button
-          text="–"
-          touchable={!disableDecrement}
-          onPress={props.onDecrement}
-          style={styleSheet.minusText}
-        />
-      </View>
+    <View style={styles.row}>
       <Button
-        text="+"
+        icon={<TextIcon code="&#xe118;" orbit={true} style={styles.icon} />}
+        touchable={!disableDecrement}
+        onPress={props.onDecrement}
+      />
+      {props.showNumber && (
+        <Text style={props.numberStyle}>
+          <Translation passThrough={props.number} />
+        </Text>
+      )}
+      <Button
+        icon={<TextIcon code="&#xe122;" orbit={true} style={styles.icon} />}
```

Comment:
I am just starting to question adding this prop `orbit={true}` everywhere. Won't it be annoying once the migration to Orbit icons is complete to remove everything? Just thinking out loud.

Reply:
Just search and replace orbit={true} -> ''

Reply:
but yeah, maybe it is time to go the extra mile and just remove the old icons

Reply:
There's only 92 files to go through... 🍰